data/sql/updates/pending_db_world/rev_1601269291015416700.sql

```diff
@@ -0,0 +1,92 @@
+
+-- Create referenced tokens into the reference table
+DELETE FROM `reference_loot_template` WHERE `Entry`=34382;
+INSERT INTO `reference_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`) VALUES 
+('34382', '40625', '0', '0', '0', '1', '1', '1', '1'),
+('34382', '40626', '0', '0', '0', '1', '1', '1', '1'),
+('34382', '40627', '0', '0', '0', '1', '1', '1', '1');
+
+-- Insert reference loot into the creature loot temlate
+DELETE FROM `gameobject_loot_template` WHERE `Entry`=25193 AND `Reference`=34382;
+INSERT INTO `gameobject_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`) VALUES 
+('25193', '2', '34382', '100', '0', '1', '0', '1', '2');
+
+-- Delete tokens which are separetely placed
+DELETE FROM `gameobject_loot_template` WHERE  `Entry`=25193 AND `Item`=40625;
+DELETE FROM `gameobject_loot_template` WHERE  `Entry`=25193 AND `Item`=40626;
+DELETE FROM `gameobject_loot_template` WHERE  `Entry`=25193 AND `Item`=40627;
+
+
+-- = Gluth 25 = 29417
+
+-- Create referenced tokens into the reference table
+DELETE FROM `reference_loot_template` WHERE `Entry`=34383;
+INSERT INTO `reference_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`) VALUES 
+('34383', '40625', '0', '0', '0', '1', '1', '1', '1'),
+('34383', '40626', '0', '0', '0', '1', '1', '1', '1'),
+('34383', '40627', '0', '0', '0', '1', '1', '1', '1'),
+('34383', '40634', '0', '0', '0', '1', '1', '1', '1'),
+('34383', '40635', '0', '0', '0', '1', '1', '1', '1'),
+('34383', '40636', '0', '0', '0', '1', '1', '1', '1'),
+('34383', '40637', '0', '0', '0', '1', '1', '1', '1'),
+('34383', '40638', '0', '0', '0', '1', '1', '1', '1'),
+('34383', '40639', '0', '0', '0', '1', '1', '1', '1');
+
+-- Insert reference loot into the creature loot temlate
+DELETE FROM `creature_loot_template` WHERE `Entry`=29417 AND `Reference`=34383;
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`) VALUES 
+('29417', '2', '34383', '100', '0', '1', '0', '1', '2');
+
+-- Delete tokens which are separetely placed
+DELETE FROM `creature_loot_template` WHERE  `Entry`=29417 AND `Item`=40625;
+DELETE FROM `creature_loot_template` WHERE  `Entry`=29417 AND `Item`=40626;
+DELETE FROM `creature_loot_template` WHERE  `Entry`=29417 AND `Item`=40627;
+DELETE FROM `creature_loot_template` WHERE  `Entry`=29417 AND `Item`=40634;
+DELETE FROM `creature_loot_template` WHERE  `Entry`=29417 AND `Item`=40635;
+DELETE FROM `creature_loot_template` WHERE  `Entry`=29417 AND `Item`=40636;
+DELETE FROM `creature_loot_template` WHERE  `Entry`=29417 AND `Item`=40637;
+DELETE FROM `creature_loot_template` WHERE  `Entry`=29417 AND `Item`=40638;
+DELETE FROM `creature_loot_template` WHERE  `Entry`=29417 AND `Item`=40639;
```

Comment:
You can do all of these in one delete

... AND Item IN (x,y,z)

Reply:
```sql

-- Delete tokens which are separetely placed
DELETE FROM `creature_loot_template` WHERE  `Entry`=29417 AND `Item` IN (40625, 40626, 40627, 40634, 40635, 40636, 40637, 40638, 40639);